### ISSUE TITLE
Test failure from tarball due to missing files

### ISSUE THREAD

**Issue by DanBurton** [henrylaxen/bbdb#1]
As seen on the stackage build server:

```
> /tmp/stackage-build12/bbdb-0.6$ dist/build/bbdb-tests/bbdb-tests
bbdb-tests: test/sampleData.txt: openFile: does not exist (No such file or directory)
```

This sort of thing is usually fixed by declaring your test files in the `extra-source-files` section of the .cabal file.

**Comment by DanBurton** [comment]
Here is how I reproduced the issue locally:

```bash
$ stack unpack bbdb-0.6 && cd bbdb-0.6
$ stack init --resolver nightly-2017-12-09
$ stack test
```

I've marked bbdb as an expected test failure in stackage. If you're able to correct the issue, then feel free to open a PR reverting this commit: https://github.com/fpco/stackage/commit/6b300dc051ee30c065811fca46e5b262bd65e3fa.

**Comment by henrylaxen** [comment]
Hi Dan,
Thanks for your (amazingly) quick response, but I'm new to github/travis/stackage and I'm not sure what your mean by "feel free to open a PR reverting this commit: fpco/stackage@6b300dc."  I've updated the bbdb.cabal file as you suggested and uploaded it to hackage.  It is now bbdb-0.6.1.  I unpacked the gz file locally and confirmed that the above code (stack init ..... stack test) runs correctly now, but I'm not sure what you want me to do next.  Sorry for all the trouble.
Best wishes,
Henry Laxen
